### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by iykon**
Correct some grammar and spelling issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-incomplete k nearest neighbor query in postgresql using LP algorithm
+Incomplete k-nearest neighbor query in postgresql using LP algorithm
 
 ## Algorithm Discription
 ### LP algorithm:
   Please refer to Dr. Gao's paper: ***IkNN-TFS-Yunjun Gao-20150115***
 ### Initialization:
   1. Set up an extra table to record lattice-bucket relations. Each Lattice is identified by `ncomplete`, the number of completed fields of all the objects in it. Each bucket is identified by a bitmap, representing the incomplete state of objects in it. e.g. '1010' represents the 2nd and 4th field of the object is incomplete.
-  2. Categorize all objects into buckets. Build a table for each bucket, storing the whole tuple and an extra column for alphavalue. A BTREE index is built on each bucket at the column alphavalue.
+  2. Categorize all objects into buckets. Build a table for each bucket, storing the whole tuple and an extra column for alphavalue. A BTREE index is built on each bucket at the column of alphavalue.
```
Comment (Travelinglight) [inline README.md]: This is actually "at the column of which the name is alphavalue". `alphavalue` is not a table.

I would suggest using
```
at the column `alphavalue`
```

File: README.md [hunk 2]
```diff
@@ -53,21 +53,21 @@ Now initialize the "test" table
 
 The lpinit function automatically does these things:
 
-  1. create a tmp table as lattices. The name of tmp table is [table name]_latmp;
-  2. add four columns to the original table: lp_id, alphavalue, nincomplete and ibitmap, recording the unique id for lp algorithm, alpha value of the entry, the number of incomplete values and the bitmap of completeness respectively;
-  3. build up hash index on ibitmap, which is for buckets deletion;
```
Comment (Travelinglight) [inline README.md:58]: Nice change!

File: README.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ This function automatically does these things:
   ~~~
 
 ### lpinit function doesn't exist after withdraw
-  In the withdraw function I dropped the lpinit function. this is convenient for me because this project is not yet finished and I need to update lpinit function frequently. All you have to do is to re-import the LPinit module.
```
Comment (Travelinglight) [inline README.md:130]: good catch.

File: README.md (hunk 1 above)
Comment (iykon) [reply README.md]: That was what I meant. I agree that adding '' is clearer.

**Review by Travelinglight — APPROVED** [review]

**Pull request merged**